torchdynamo/debug_utils.py

```diff
@@ -404,24 +404,27 @@ def deferred_for_real_inputs(real_inputs):
                     raise NotImplementedError(
                         "Accuracy minification is supported for inductor only"
                     )
+                cloned_real_inputs = [x.clone() for x in real_inputs]
```

Comment:
you can just clone the list - don't need to clone the inputs. `x.clone()` won't preserve strides (or use clone_preserve_strides or whatever it's called)

`list(real_inputs)`

Reply:
Good catch!